src/Serialization/Address.js

```diff
@@ -0,0 +1,63 @@
+const CardanoWasm = require("@emurgo/cardano-serialization-lib-nodejs");
+
+
+// newBaseAddress :: NetworkId -> PubKeyHash -> StakeKeyHash -> BaseAddress
+exports.newBaseAddress = netId => pkhBech32 => skhBech32 => {
```

Comment:
This is also partial, so I think that we should reflect that in the type signature. `from_bech32()` might throw several different exceptions if the input if malformed (we don't have a smart constructor for `Bech32String` or another guarantee that it will be correct). We could just make this an `Effect` and do away with the checks in JS, but you took the approach of returning `Maybe`s in some implementations below, which I think is fine, so maybe we can do that here too

Perhaps a similar approach to the one you take in `fromBech32Impl` would work here, e.g.

```js
// newBaseAddress :: 
//   (forall a. a -> Maybe a)
//   (forall a. Maybe a)
//   NetworkId -> 
//   PubKeyHash -> 
//   StakeKeyHash -> 
//   Maybe BaseAddress
exports.newBaseAddress =
  just => nothing => netId => pkhBech32 => skhBech32 => {
    try {
      const pkh = CardanoWasm.Ed25519KeyHash.from_bech32(pkhBech32);
      const skh = CardanoWasm.Ed25519KeyHash.from_bech32(skhBech32);
      // null checks... (what if both are null?)
      const baseAddr = CardanoWasm.BaseAddress.new(
        netId,
        CardanoWasm.StakeCredential.from_keyhash(pkh),
        CardanoWasm.StakeCredential.from_keyhash(skh)
      );
      return just(baseAddr);
    } catch (err) {
      // maybe log it
      return nothing;
    }
  };
```